web/src/locales/zh-CN/menu.ts

```diff
@@ -22,7 +22,7 @@ export default {
   'menu.login': '登录',
   'menu.register': '注册',
   'menu.register.result': '注册结果',
-  'menu.dashboard': 'Dashboard',
+  'menu.dashboard': '仪表板',
```

Comment:
```suggestion
  'menu.dashboard': 'Dashboard',
```

Reply:
may I ask why we keep `Dashboard`  in zh-CN? @juzhiyuan